src/components/submission-common.js

```diff
@@ -338,6 +338,14 @@ const validateRequirements = (inputSource, inputType) => {
       return _.every(Boolean, fieldsValidated)
     }
     if (inputSource.type === 'literal') {
+      // this condition is specifically added to allow '0' and 'false' as valid values because
+      // '!!inputSource.parameter_value' considers '0' and 'false' as empty values
+      // Note: '!=' null check also covers if value is undefined
+      if (inputSource.parameter_value != null &&
+        (inputSource.parameter_value === 0 || inputSource.parameter_value === false)) {
+        return true
+      }
+
       return !!inputSource.parameter_value
```

Comment:
is this all just a long winded way of saying `parameter_value != null` now?

Reply:
`!!inputSource.parameter_value` also verifies that the value is not empty string `""` or NaN hence I left it as it there